app/views/index.erb

```diff
@@ -0,0 +1,49 @@
+<div id="index_background">
+
+</div>
+
+<div id="header">
+  <h1 class="index">The Command Tower</h1>
+  <p>Placeholder Text</p>
+</div>
+
+<div id="login">
+  <p>View your libraries</p>
+  <form action="/login" method="post">
+    <input type="text" name="username" placeholder="Username">
+    <br>
```

Comment:
using `<br>` tags for spacing is considered a bad practice. Use `div`s, classnames and CSS instead :)